userguide/system.rst

```diff
@@ -2179,8 +2183,8 @@ complete the following fields:
   categories are populated to the drop-down menu. Select the one that
   best describes the bug or feature being reported.
 
-* **Attach Debug:** it is recommended to check this box so that an
-  overview of the system's hardware, build string, and configuration is
+* **Attach Debug:** it is recommended to enable this option so that an
```

Comment:
Gah, "it is recommended".

enabling this option is recommended so an overview...